DelayLotkaVolterra/VolterraExp.jl

```diff
@@ -8,122 +8,107 @@ using Pkg; Pkg.activate("."); Pkg.instantiate()
 using OrdinaryDiffEq
 using ModelingToolkit
 using DataDrivenDiffEq
-using Flux, Tracker
-using LinearAlgebra
-using DiffEqFlux
+using LinearAlgebra, DiffEqSensitivity, Optim
+using DiffEqFlux, Flux
 using Plots
 gr()
 
-
-
 function lotka(du, u, p, t)
     α, β, γ, δ = p
     du[1] = α*u[1] - β*u[2]*u[1]
     du[2] = γ*u[1]*u[2]  - δ*u[2]
 end
 
 # Define the experimental parameter
-tspan = (0.0f0,3.0f0)
-u0 = [0.44249296,4.6280594]
-p = Float32[1.3, 0.9, 0.8, 1.8]
-prob = ODEProblem(lotka, u0,tspan, p)
+tspan = (0.0f0,2.0f0)
+u0 = Float32[0.44249296,4.6280594]
+p_ = Float32[1.3, 0.9, 0.8, 1.8]
```

Comment:
I had to rename this, since it was reused in the NN model. Somehow the NN parameters ended up as the death rates of the predator and prey.